src/components/restClient/HistoryPanel.tsx

```diff
@@ -0,0 +1,5 @@
+import Title from 'antd/es/typography/Title';
+

```

Comment:
мб лучше так доставать Title
```suggestion
import { Typography } from 'antd';

const { Title } = Typography;
```

вдруг мы захотим ещё какой-то метод из типографии достать

Reply:
Вдруг мы удалим эту панель😕